### ISSUE TITLE
Debuffs should not be removed upon leaving & entering battlegrounds/arenas

### ISSUE THREAD

**Issue by Klimpb** [ccshiro/cc-buglist#1404]
http://us.battle.net/wow/en/forum/topic/4138434536

Can find videos as proof too. I remember playing back in TBC/Wotlk and having a lock use curse of elements on me in one game and I still had it on me when I joined the next game. Though most of the time the debuff timer ran out in the prep room.

**Comment by ccshiro** [comment]
Yeah, I too remember this. Also it works like this currently on retail, and I doubt it's something they'd introduce if it wasn't like this from the start.

**Comment by Klimpb** [comment]
Any 2.4.3 website that lets me list all debuffs in the game that lasts longer than 30 seconds or so?
Blizzard made some debuffs get removed when entering arena, a few debuffs which could be obtained from mobs that lasted a minute or so and granted attack speed or some other benficial stat.

Should probably also remove all debuffs from pets that you summon for the first time in arena to avoid things like this: https://www.youtube.com/watch?v=wLVCkeisoRo

Debuffs off the top of my head that should be removed upon entering arena:
-Curse of Recklessness
